crates/forge/tests/e2e/running.rs

```diff
@@ -806,6 +808,26 @@ fn validate_init(temp: &TempDir, validate_snforge_std: bool) {
     assert_stdout_contains(output, expected);
 }
 
+fn validate_gitignore(temp: &TempDir) {
+    let gitignore_path = temp.join("test_name/.gitignore");
+    let gitignore_content = fs::read_to_string(gitignore_path).unwrap();
+
+    let expected_entries = vec![
+        ".snfoundry_cache/",
+        "snfoundry_trace/",
+        ".snfoundry_versioned_programs/",
+        "coverage/",
+    ];
+
+    for entry in expected_entries {
+        assert!(
+            gitignore_content.contains(entry),
+            "Expected .gitignore to contain '{}', but it didn't",
+            entry
+        );
+    }
+}
```

Comment:
I think you should be able to get rid of this `for` loop if you used `indoc!` to hold expected entries. 
hint: you can try and compare the whole file

Reply:
resollved